util/pkg/mirrors/mirrors.go

```diff
@@ -39,15 +39,22 @@ func FindUrlMirrors(u string) []string {
 
 	var mirrors []string
 	if strings.HasPrefix(u, baseURLString) {
+
 		suffix := strings.TrimPrefix(u, baseURLString)
 		// artifacts.k8s.io is the official and preferred mirror.
 		mirrors = append(mirrors, fmt.Sprintf(kubernetesKopsMirrorBase, kops.Version)+suffix)
 		// GitHub artifact names are quite different, because the suffix path is collapsed.
 		githubSuffix := strings.ReplaceAll(suffix, "/", "-")
 		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-nodeup", "nodeup-linux-amd64")
 		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-nodeup", "nodeup-linux-arm64")
+		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-protokube", "protokube-linux-amd64")
+		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-protokube", "protokube-linux-arm64")
+		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-channels", "channels-linux-amd64")
+		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-channels", "channels-linux-arm64")
+
 		mirrors = append(mirrors, fmt.Sprintf(githubKopsMirrorBase, kops.Version)+githubSuffix)
 	}
+
```

Comment:
Nit:
```suggestion
		suffix := strings.TrimPrefix(u, baseURLString)
		// artifacts.k8s.io is the official and preferred mirror.
		mirrors = append(mirrors, fmt.Sprintf(kubernetesKopsMirrorBase, kops.Version)+suffix)
		// GitHub artifact names are quite different, because the suffix path is collapsed.
		githubSuffix := strings.ReplaceAll(suffix, "/", "-")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-nodeup", "nodeup-linux-amd64")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-nodeup", "nodeup-linux-arm64")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-protokube", "protokube-linux-amd64")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-protokube", "protokube-linux-arm64")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-amd64-channels", "channels-linux-amd64")
		githubSuffix = strings.ReplaceAll(githubSuffix, "linux-arm64-channels", "channels-linux-arm64")
		mirrors = append(mirrors, fmt.Sprintf(githubKopsMirrorBase, kops.Version)+githubSuffix)
	}
```

Reply:
Don't see any difference apart from a space. will remove the space

Reply:
yup, only removal of extra lines.